pando-rt/src/wait.cpp

```diff
@@ -142,6 +87,27 @@ void waitAll() {
 #ifdef PANDO_RT_ENABLE_MEM_STAT
   MemTraceStat::writePhase();
 #endif
+#elif defined(PANDO_RT_USE_BACKEND_DRVX)
+  CommandProcessor::barrier();
+  for (std::int64_t i = 0; i < Drvx::getNodeDims().id; i++) {
+    for (std::int64_t j = 0; j < Drvx::getPodDims().x; j++) {
+      while (DrvAPI::getPodTasksRemaining(i, j) != 0) {
+        hartYield();
```

Comment:
is this supposed to be this hartYield? or do we want NOP(1u) or smthing.

Reply:
This is supposed to be hartYield. You are waiting for all cores task queues to be empty. This will likely be a long wait for the first core to reach this function. Instead of checking back every 1 cycle for the pioneer, you probably want to check back every other 1000 cycles.